modules/highgui/include/opencv2/highgui.hpp

```diff
@@ -863,4 +863,6 @@ CV_EXPORTS int createButton( const String& bar_name, ButtonCallback on_change,
 
 } // cv
 
+#include "highgui/viz3d.hpp"
```

Comment:
I propose to not include 3d part by default:
- It may affect user code. It does not expect that GLFW in OpenCV.
- The API will be experimental for some time and may change.